client/src/components/Header.css

```diff
@@ -2,18 +2,27 @@ body {
     font-family: "Raleway", sans-serif;
     font-weight: 400;
     line-height: 1.666;
-    background: #ffffff;
-    color: #555;
+    /* background: #ffffff;
+    color: #555; */
   }
   
   .header-section{
-    background-color:#efefef;
+    /* background-color:#efefef; */
     padding: 2px;
     display: flex;
     flex-direction: row;
      align-items: center;
     justify-content: space-between;
   }
+
+  .imageBanner---h3G2 {
```

Comment:
`imageBanner---h3G2` is a strange classname. This could simply be 'image-banner'